pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -653,6 +709,9 @@ func (p *criStatsProvider) getAndUpdateContainerUsageNanoCores(stats *runtimeapi
 	if stats == nil || stats.Attributes == nil || stats.Cpu == nil || stats.Cpu.UsageCoreNanoSeconds == nil {
 		return nil
 	}
+	if stats.Cpu.UsageNanoCores != nil {
```

Comment:
ditto here, worth to explain that cache is unused

Reply:
done!